plugins/code-versions/src/utils/shared-styles.ts

```diff
@@ -0,0 +1,8 @@
+import { cn } from "../utils"
+
+/**
+ * One-time fade in animation that runs only once when the element is first rendered
+ *
+ * The animation starts with opacity 0 and animates to opacity 1 over 150ms
+ */
+export const fadeInAnimationClassName = cn("animate-[fadeIn_150ms_forwards]")
```

Comment:
Hmm, is there no Tailwind-native way to do this? The `cn()` hack feels easy to forget 🙈

Reply:
Yeah, I understand why `cn` is there, that's what I meant by "for it to work". But there must be some `@variant` or something you can do? Then you don't need to import, don't need to wrap other `className` in `cn`, more idiomatic.

Reply:
as discussed offline: at a later stage, we should collect the arbitrary values. Then, this constant becomes obsolete.